### PR TITLE
Partial edit. I modified the first few sections.

### PR DESCRIPTION
**Pull request by robblovell**
Partial edit to clarify and excite!

### CODE REVIEW

File: content/guides/firstapp.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
 
-* Applications should be simple to set up and use. Provisioners should hide the complexity of application installation, removal and configuration. Documentation should avoid the use of jargon where possible.
+* Applications should be simple to set up and use. Provisioners should hide the complexity of application installation, removal and configuration. Documentation should avoid the use of jargon and acronyms where possible.
 
-* Applications should be well defined. Applications should not install more or less than is required for the application. If Applications require other components or other Applications to extend functionality, they should use Application Linking when possible.  A common example of this might be adding additional functionality for for logging or metrics.
+* Applications should be well defined. Applications should not install more or less than is required for the application. If Applications require other components or other Applications to extend functionality, they should use Application Linking when possible. A common example of this might be for logging or metrics.
```
Comment (robblovell) [inline content/guides/firstapp.md:21]: We should describe what "Application Linking" is.

File: content/guides/firstapp.md [hunk 2]
```diff
@@ -1,24 +1,26 @@
 # Your First Application
 
-Any existing Kubernetes application can become a CodeZero Application. Unlike Deployments, Services and PersistenVolumes, Applications are not a first-class resource type in Kubernetes.
+Any existing Kubernetes *application* can become a *CodeZero Application*. Unlike Kubernetes *deployments*, *services* and *Persistent Volumes*, *applications* are not a first-class resource type in Kubernetes. CodeZero treats applications as first class resources within Kubernetes so that they can become *Cloud Applications* in any cloud where Kubernetes is available.
 
-In order to publish an Application, you need to:
+In order to turn any Kubernetes application into a *Cloud Application*, the application needs to be published to CodeZero's Marketplace so that it can be installed and used by end users using the CodeZero operating system to access them in the cloud.
```
Comment (nsainaney) [inline content/guides/firstapp.md:5]: Just the manifest. Your application is still published to Docker and the provisioner is published to npm

**Review by nsainaney — CHANGES_REQUESTED** [review]

File: content/guides/firstapp.md [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
-In order to publish an Application, you need to:
+In order to turn any Kubernetes application into a *Cloud Application*, only the application's manifest needs to be published to CodeZero's Marketplace so that it can be installed and used by end users using the CodeZero operating system to access them in the cloud. (Your application, and other information about, it will be picked up from the docker and npm registries where it is publically available).
 
-1. Create an Application Manifest that defines the parameters of your Application
-1. Write a Provisioner for your Application
-1. Register your Application on the [CodeZero Hub](https://hub.codezero.io)
+To become a Cloud Application, you need to:
+
+1. Create a *Cloud Application Manifest* that defines the parameters of your application
+1. Write a *CodeZero Provisioner* for your application that will turn it into a CodeZero available *Cloud Application*
```
Comment (conneryn) [inline content/guides/firstapp.md:10]: Should we be explicit that the provisioner needs to be published to NPM?

**Review by conneryn — APPROVED** [review]

File: content/guides/firstapp.md (hunk 3 above)
Comment (robblovell) [reply content/guides/firstapp.md:10]: yes..

File: content/guides/firstapp.md [hunk 4]
```diff
@@ -4,10 +4,17 @@ Any existing Kubernetes *application* can become a *CodeZero Application*. Unlik
...
+
+1. A working application that is well tested
+1. The applicatoin runs inside a docker container with a Dockerfile and is published to Docker Hub
+1. The application runs inside of a Kubernetes cluster and has a Kubernetes CRD
+
 To become a Cloud Application, you need to:
 
 1. Create a *Cloud Application Manifest* that defines the parameters of your application
 1. Write a *CodeZero Provisioner* for your application that will turn it into a CodeZero available *Cloud Application*
+1. Publish your *Cloud Application Manifest* to NPM
```
Comment (conneryn) [inline content/guides/firstapp.md:17]: The Provisioner package (not the application manifest, at least, not yet), gets published to NPM.

**Comment by conneryn** [comment]
Removed myfirstapp, in favour of a AppEngine walkthrough.

**Pull request closed without merging**